### PR TITLE
Make "Remove an intent" cluster-friendly

### PR DESCRIPTION
**Pull request by helmste**
Added a script that would remove existing intents, intent-based virtual switches and QoS policies from all nodes of a cluster to allow for successful re-deployment attempt.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@helmste : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/hci/manage/manage-network-atc.md [hunk 1]
```diff
@@ -129,19 +129,50 @@ This task will help you override the default configuration which has already bee
 
 If you want to test various configurations on the same adapters, you may need to remove an intent. 
 
-If you previously deployed and configured Network ATC on your system, you may need to reset the node so that the configuration can be deployed. To do this, copy and paste the following commands to remove all existing intents and their corresponding vSwitch:
+If you previously deployed and configured Network ATC on your system, you may need to reset the cluster or node so that the configuration can be deployed. To do this on a cluster-based deploynent, copy and paste the following commands to one cluster node to remove all existing intents and their corresponding vSwitch:
```
Comment (dcuomo) [inline azure-stack/hci/manage/manage-network-atc.md:132]: I don't like the terminology of "reset the cluster" which I think gives the wrong impression. Consider: "you may need to remove the existing configuration as Network ATC will not remove them for you." or something to that effect.

" to one cluster node" should be "on one cluster node"

**Review by dcuomo — CHANGES_REQUESTED** [review]
Hi @helmste - minor wording changes for customer clarity. Otherwise LGTM. Thanks for writing this up!

**Comment by PRMerger12** [comment]
@helmste : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JasonGerend — APPROVED** [review]

**Comment by JasonGerend** [comment]
#sign-off

**Review by JasonGerend — COMMENTED** [review]
@helmste  Here are my suggested edits from the email thread. I can't edit the heading on line 128 from here, but my suggested heading is: Remove all intents and start over

**Comment by PRMerger14** [comment]
@helmste : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@helmste : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@helmste : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@JasonGerend Could you confirm that you're ready for the PR to be merged? Thanks.

**Review by JasonGerend — APPROVED** [review]

**Comment by JasonGerend** [comment]
#sign-off. Yes please!

**Pull request merged**